app/src/main/java/com/arpitkatiyarprojects/countrypickerproject/ui/previews/PreviewHelper.kt

```diff
@@ -0,0 +1,29 @@
+package com.arpitkatiyarprojects.countrypickerproject.ui.previews
+
+import android.content.res.Configuration
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+import com.arpitkatiyarprojects.countrypickerproject.ui.theme.CountryPickerProjectTheme
+
+/** Apply theme and background- / contentColor to the content of every Preview which uses this composable */
+@Composable
+fun MyPreview(
+    content: @Composable () -> Unit,
+) {
+    CountryPickerProjectTheme {
+        Surface {
+            content()
+        }
+    }
+}
```

Comment:
Shows light/dark mode colors in previews properly, using the theme as base.

Not important for this PR, as colours don't affect this issue fixed here. However, I added this directly as base for future PRs where it does matter.